crates/chain/tests/test_tx_graph.rs

```diff
@@ -664,17 +668,17 @@ fn test_walk_ancestors() {
 
     let ancestors = [
         graph
-            .walk_ancestors(&tx_c0, |depth, tx| Some((depth, tx)))
+            .walk_ancestors(Arc::new(tx_c0.clone()), |depth, tx| Some((depth, tx)))
```

Comment:
I'm guessing this also works.

```suggestion
            .walk_ancestors(tx_c0, |depth, tx| Some((depth, tx)))
```